vmdb/app/models/service.rb

```diff
@@ -4,7 +4,6 @@ class Service < ActiveRecord::Base
   belongs_to :service_template               # Template this service was cloned from
   belongs_to :service                        # Parent Service
   has_many :services, :dependent => :destroy # Child services
-  has_many :ems_events
```

Comment:
If this side of the relationship is being removed, I'd expect to also see a migration to remove the `service_id` field from the `ems_events` table as well as an update to the `EmsEvent` model class to remove the `belongs_to :service`.